gammapy/modeling/fit.py

```diff
@@ -442,6 +461,33 @@ def stat_surface(self, datasets, x, y, reoptimize=False):
         results : dict
             Dictionary with keys "x_values", "y_values", "stat" and "fit_results".
             The latter contains an empty list, if `reoptimize` is set to False.
+
+        Examples
+        --------
+        >>> from gammapy.datasets import Datasets, SpectrumDatasetOnOff
+        >>> from gammapy.modeling.models import SkyModel, LogParabolaSpectralModel
+        >>> from gammapy.modeling import Fit
+        >>> import numpy as np
+        >>> datasets = Datasets()
+        >>> for obs_id in [23523, 23526]:
+        ...     dataset = SpectrumDatasetOnOff.read(
+        ...         f"$GAMMAPY_DATA/joint-crab/spectra/hess/pha_obs{obs_id}.fits"
+        ...     )
+        ...     datasets.append(dataset)
+        >>> datasets = datasets.stack_reduce(name="HESS")
+        >>> model = SkyModel(spectral_model=LogParabolaSpectralModel(), name="crab")
+        >>> datasets.models = model
+        >>> par_alpha = datasets.models.parameters["alpha"]
+        >>> par_beta = datasets.models.parameters["beta"]
+        >>> par_alpha.scan_values = np.linspace(1.55, 2.7, 20)
+        >>> par_beta.scan_values = np.linspace(-0.05, 0.55, 20)
```

Comment:
If this takes too long to run, you can decrease the number of points

Reply:
tried it, it's quite fast